pm4ml-experience-api/values.yaml

```diff
@@ -20,6 +20,8 @@ env:
   managementEndPoint: management-api
   dfspId: test
   metricsEndPoint: prometheus-server
+  CACHE_URL: redis://redis-master:6379
+  authSessionSecure: true
```

Comment:
I think these env variables are not going to be pulled in k8s deployment automatically. You may need to add these new parameters to pm4ml-experience-api/templates/deployment.yaml

Reply:
Alright, noted.